Tests/TuistKitTests/Mappers/FocusTargetsGraphMappersTests.swift

```diff
@@ -149,11 +149,11 @@ final class FocusTargetsGraphMappersTests: TuistUnitTestCase {
         )
     }
 
-    func test_map_when_included_targets_is_target_with_no_dependency_but_with_test_target_also_test_target_is_pruned() throws {
+    func test_map_when_included_targets_is_target_with_no_dependency_but_with_test_target_also_test_target_is_not_pruned() throws {
         // Given
         let targetNames = ["foo", "bar", "baz"].shuffled()
         let aTarget = Target.test(name: targetNames[0])
-        let aTestTarget = Target.test(name: targetNames[0] + "Tests", product: .unitTests)
+        let aTestTarget = Target.test(name: targetNames[0] + "Tests", product: .unitTests, dependencies: [.target(name: targetNames[0])])
```

Comment:
My impl looks for the dependency in the target, so I had to add this to get the updated test to pass. It looks like dependencies could be specified in two different places, please let me know if I should use a different mechanism to find dependencies.